package.json

```
@@ -135,7 +135,7 @@
 		"@elementor/query": "0.2.2",
 		"@elementor/schema": "0.1.1",
 		"@elementor/store": "0.8.5",
-		"@elementor/ui": "1.21.7",
+		"@elementor/ui": "^1.21.13",
```

```suggestion
		"@elementor/ui": "1.21.13",
```
On purpose, for not being updated automatically.